src/pages/innsending.tsx

```diff
@@ -0,0 +1,54 @@
+import { GetServerSidePropsContext } from "next";
+import React from "react";
+import ErrorPage from "./_error";
+import { startSoknad } from "./api/quiz-api";
+import { getSession } from "@navikt/dp-auth/server";
+import { audienceDPSoknad } from "../api.utils";
+
+export async function getServerSideProps(context: GetServerSidePropsContext) {
+  if (process.env.NEXT_PUBLIC_LOCALHOST) {
+    return {
+      redirect: {
+        destination: `./uuid-innsending/`,
+        permanent: false,
+      },
+    };
+  }
+
+  const { token, apiToken } = await getSession(context);
+  if (!token || !apiToken) {
+    // TODO Redirect til hvilken login?
+    return {
+      redirect: {
+        destination: "/TODO-redoratoren-login",
```

Comment:
`[base_path]/oauth2/login` er riktig URL for login nå 🚀 

Men nå skal det fryktelig mye til for å komme inn hit.

Reply:
Supert! Da endrer jeg det også de andre stedene vi har denne TODOen 🙌